help/custom-profile-fields.md

```diff
@@ -85,6 +91,21 @@ checkboxes will be disabled.
 
 {end_tabs}
 
+## Profile field types
+
+There are several different types of fields available.
+
+* **Short text**: For one line responses, like
+    "Job title". Responses are limited to 50 characters.
+* **Long text**: For multiline responses, like "Biography".
+* **Date picker**: For dates, like "Birthday".
+* **Link**: For links to websites.
+* **External account**: For linking to GitHub, Twitter, etc.
+* **Pronouns**: What pronouns should people use to refer to the user?
+* **List of options**: Creates a dropdown with a list of options.
+* **Person picker**: For selecting one or more users, like "Manager" or
+    "Direct reports".
```

Comment:
Can you split the change that is moving this to the bottom in its own prep commit with its own commit message? It's really helpful for code reviewers to do that, since then we don't unexpectedly see a block of text being removed when reviewing and need to figure out why.

(This is a very helpful general strategy for communication in a pull request review).

Reply:
I have splitted the changes. Will keep this point in mind in future PRs.